collector/container/konflux.Dockerfile

```diff
@@ -91,33 +76,14 @@ RUN ./builder/install/install-dependencies.sh && \
     ctest -V --test-dir ${CMAKE_BUILD_DIR} && \
     strip -v --strip-unneeded "${CMAKE_BUILD_DIR}/collector/collector"
 
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
* Try with ubi-minimal

Reply:
* try with ubi 9 to see if we can support very different repos,
* RUN cat /cachi2/... fetched content